models/oneinch/ethereum/oneproto/oneinch_oneproto_ethereum_trades.sql

```diff
@@ -113,65 +115,68 @@ SELECT
         *
         (
             CASE
-                WHEN token_sold_address = '{{burn_address}}'
+                WHEN token_sold_address = {{burn_address}}
                     THEN prices_eth.price
                 ELSE prices_sold.price
             END
         )
     ) AS amount_usd
     ,src.token_bought_address
     ,src.token_sold_address
-    ,coalesce(src.taker, tx.from) AS taker
+    ,coalesce(src.taker, tx."from") AS taker
     ,src.maker
     ,src.project_contract_address
     ,src.tx_hash
-    ,tx.from AS tx_from
+    ,tx."from" AS tx_from
     ,tx.to AS tx_to
-    ,CAST(src.trace_address as array<long>) as trace_address
+    ,src.trace_address
     ,src.evt_index
 FROM
     oneinch as src
-LEFT ANTI JOIN --where tx_hash isn't already pulled from oneinch version spells
-    (
-        SELECT DISTINCT
-            tx_hash
-            , block_number
-        FROM
-            {{ ref('oneinch_v1_ethereum_trades') }}
-        UNION ALL
-        SELECT DISTINCT
-            tx_hash
-            , block_number
-        FROM
-            {{ ref('oneinch_v2_ethereum_trades') }}
-        UNION ALL
-        SELECT DISTINCT
-            tx_hash
-            , block_number
-        FROM
-            {{ ref('oneinch_v3_ethereum_trades') }}
-        UNION ALL
-        SELECT DISTINCT
-            tx_hash
-            , block_number
-        FROM
-            {{ ref('oneinch_v4_ethereum_trades') }}
-        UNION ALL
-        SELECT DISTINCT
-            tx_hash
-            , block_number
-        FROM
-            {{ ref('oneinch_v5_ethereum_trades') }}
-    ) oneinch
-    ON src.tx_hash = oneinch.tx_hash
-    AND src.block_number = oneinch.block_number
+{#
+-- commenting out to test gh run since those models haven't been migrated
+-- LEFT ANTI JOIN --where tx_hash isn't already pulled from oneinch version spells
```

Comment:
i also believe we will need to switch from `left anti join` to `left join where rows returned are null`

Reply:
oke ser!